src/idlcxx/src/generator.c

```diff
@@ -746,6 +749,7 @@ const char *bnd_str_inc = "<string>";
 const char *uni_tmpl = "std::variant";
 const char *uni_get_tmpl = "std::get";
 const char *uni_inc = "<variant>";
+const char *swap_inc = "<utility>";
```

Comment:
I think I'm missing the addition of a flag that allows the user to specify a swap include?

Reply:
Since I did not implement a user-settable swap function yet, this was added just for generating header files when a swap was encountered.
I will extend the settability of user-defined functions with a swap function.